Runtime/Action/AllAction.cs

```diff
@@ -63,7 +63,10 @@ protected virtual void RemoveActionsListeners()
 
             foreach (Action action in Actions.SubscribableElements)
             {
-                action.ActivationStateChanged.RemoveListener(OnActionActivationStateChanged);
+                if (action)

```

Comment:
Code convention: We want to be explicit about what defines the "boolness" of objects, especially since [Unity overrides the equality of `UnityEngine.Object`s](https://blogs.unity3d.com/2014/05/16/custom-operator-should-we-keep-it/).
```suggestion
                if (action != null)
```

Reply:
should i make the actual change rather than just pressing the resolve conversation button?

Reply:
Doesn't matter how you apply the suggestions! All that matters in the end is that you squash the commits into one, leaving the nice subject and body for the message, explaining the change (and the "why").